processor/transformer/transformer.go

```diff
@@ -51,7 +51,6 @@ type MetadataT struct {
 	SourceTpConfig      map[string]map[string]interface{} `json:"sourceTpConfig"`
 	MergedTpConfig      map[string]interface{}            `json:"mergedTpConfig"`
 	DestinationID       string                            `json:"destinationId"`
-	JobRunID            string                            `json:"jobRunId"`
```

Comment:
any idea why this was there? @chandumlg ?

Reply:
Not sure. Looks like a left over. 

I don't see any value set to that variable. No point keeping it.